### PR TITLE
Update Product Summary README.md

### PR DESCRIPTION
**Pull request by guerreirobeatriz**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,60 +1,68 @@
-📢 Don't fork this project. Use, [contribute](https://github.com/vtex-apps/awesome-io#contributing), or open issues through [Store Discussion](https://github.com/vtex-apps/store-discussion).
+📢 Use this project, [contribute](https://github.com/vtex-apps/breadcrumb) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
```
Comment (victorhmp) [inline docs/README.md]:
```suggestion
📢 Use this project, [contribute](https://github.com/vtex-apps/product-summary) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
```

File: docs/README.md [hunk 2]
```diff
@@ -1,60 +1,68 @@
...
 
 # Product Summary
 
-Product Summary summarises the product informations such as name, price and picture.
-This is a VTEX app that is used by store theme.
+Product Summary is an app responsible for summarizing product information (such as name, price and image) in other store blocks, such as the [Shelf](https://vtex.io/docs/components/all/vtex.shelf/) and the [Minicart](https://vtex.io/docs/components/all/vtex.minicart/).
```
Comment (victorhmp) [inline docs/README.md]: The `product-summary` is not used in the minicart anymore :)
It was used in the legacy one.

File: docs/README.md [hunk 3]
```diff
@@ -1,60 +1,68 @@
...
 
 ![image](https://user-images.githubusercontent.com/284515/70235170-1a503a80-1741-11ea-952d-07b178995f92.png)
 
 ## Configuration
 
-1. Import the vtex.store-component's app to your theme's dependencies in the manifest.json, for example:
+1. Import the `vtex.product-summary` app to your theme's dependencies in the `manifest.json`:
 
 ```json
   dependencies: {
```
Comment (victorhmp) [inline docs/README.md]:
```suggestion
  "dependencies": {
```

File: docs/README.md [hunk 4]
```diff
@@ -1,60 +1,68 @@
...
+
+
+2. Add the `product-summary.shelf` block to the block that will host the product information, such as the Shelf. Notice that although the block's name refers to the Shelf block, it can and should be used in any block that is able to render product information, such as the Minicart and those found on the [Search Results](https://vtex.io/docs/components/all/vtex.search-result/) page.
+
+```json
+    "shelf#home": {
+    "blocks": [
+      "product-summary.shelf"
+    ],
+
```
Comment (victorhmp) [inline docs/README.md]:
```suggestion
    }
```

File: docs/README.md [hunk 5]
```diff
@@ -1,60 +1,68 @@
...
+    "children": [
+      "product-summary-name",
+      "product-summary-description"
+      "product-summary-image",
+      "product-summary-price",
+      "product-summary-sku-selector",
+      "product-summary-buy-button"
+    ]
   }
 }
```
Comment (victorhmp) [inline docs/README.md]:
```json
{
  "shelf#home": {
    "blocks": ["product-summary.shelf"]
  },

  "product-summary.shelf": {
    "children": [
      "product-summary-name",
      "product-summary-description",
      "product-summary-image",
      "product-summary-price",
      "product-summary-sku-selector",
      "product-summary-buy-button"
    ]
  }
}
```

File: docs/README.md (hunk 1 above)
Comment (guerreirobeatriz) [inline docs/README.md]: 📢 Use this project, contribute to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).

**Pull request merged**